src/renderer/helpers/api/local.js

```diff
@@ -63,6 +64,14 @@ async function createInnertube({ withPlayer = false, location = undefined, safet
 let searchSuggestionsSession = null
 
 export async function getLocalSearchSuggestions(query) {
+  const searchCharLimit = store.getters.getSearchCharacterLimit
+
+  if (query.length > searchCharLimit) {
+    // There's an event handler on the search input so avoid displaying an exception
+    console.error('Query is over ' + searchCharLimit + ' characters')
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
```suggestion
    console.error(`Query is over ${searchCharLimit} characters`)
```

Reply:
Done